modules/op-release-notes-1-9.adoc

```diff
@@ -373,3 +373,42 @@ spec:
 * Before this update, a pull request failed if the user in the annotation provided values by using a regex form, for example, `refs/head/rel-*`. The pull request failed because it was missing `refs/heads` in its base branch. With this update, the prefix is added and checked that it matches. This resolves the issue and the pull request is successful.	
 // Savita Ashture
 
+[id="release-notes-1-9-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.1
+
+With this update, {pipelines-title} General Availability (GA) 1.9.1 is available on {product-title} 4.12.
+
+[id="fixed-issues-1-9-1_{context}"]	
+== Fixed issues	
+
+* Before this update, the `tkn pac repo list` command did not run on Microsoft Windows. With this update, this issue is resolved and now you can run the `tkn pac repo list` command on on Microsoft Windows.
+
+* Before this update, {pac} watcher did not receive all of the configuration change events. With this update, this issue is resolved. The {pac} watcher is updated and does not miss the configuration change events.
+
+* Before this update, TaskRuns or PipelineRuns pods created by {pac} could not access custom certificates exposed by the user in the cluster. With this update, this issue has been resolved. You can now access custom certificates in the cluster from TaskRuns or PipelineRuns pods.
+
+* Before this update, the core interceptor `tekton-triggers-core-interceptors` used in the `Trigger` resource did not function after the Pipelines operator was upgraded to version 1.9. With this update, this issue is resolved. The core interceptor, `tekton-triggers-core-interceptors`, has been updated to TLS version 1.2, which ensures that the functionality of the core interceptor runs accurately.
+
+* Before this update, when using a pipeline run with an internal OpenShift image registry, the URL to the image was hardcoded in the pipeline run definition. For example:
++
+[source,yaml]
+----
+...
+  - name: IMAGE_NAME
+    value: 'image-registry.openshift-image-registry.svc:5000/<test_namespace>/<test_pipelinerun>'
+...
+----
++
+When using in the context of {pac}, such hardcoded values prevents pipeline run definitions from being used in different clusters and namespaces.
++
+With this update, you can generalize the pipeline run definitions by using the dynamic template variables, instead of hardcoding the values for namespaces and pipeline run names. For example:
++
+[source,yaml]
+----
+...
+  - name: IMAGE_NAME
+    value: 'image-registry.openshift-image-registry.svc:5000/{{ namespace }}/{{ pipeline_run_name }}'
+...
+----
+
+* With this update, {pac} uses the same GitHub token to fetch a remote task available in the same host on GitHub.
```

Comment:
This is related to this 1.9.0 RN item 

```
Before this update, the GitHub apps token scoping was not defined and tokens would be used on every repository installation. With this update, you can scope the GitHub apps token to the target repository using the following parameters:

secret-github-app-token-scoped: Scopes the app token to the target repository, not to every repository the app installation has access to.

secret-github-app-scope-extra-repos: Customizes the scoping of the app token with an additional owner or repository.
```

In 1.9.0 it worked only with default git branch, now it works with any branch

Reply:
Fixed.